skpy/event.py

```diff
@@ -103,9 +103,13 @@ class SkypeEndpointEvent(SkypeEvent):
             Name of the device connected with this endpoint.
         capabilities (str list):
             Features available on the device.
+        version (str):
+            number of skype version 
+        typ (int):
```

Comment:
I'd rather this was called `type` in full -- attributes in SkPy objects prefer readability over mapping to protocol fields.